catalog/spatial/geocoding/spatial-geocoding-localsearch/src/main/java/org/codice/ddf/spatial/geocoder/geonames/GeoNamesLocalIndex.java

```diff
@@ -56,4 +57,14 @@ public GeoResult getLocation(final String location) {
 
         return null;
     }
+
+    public NearbyLocation getNearbyCity(String location) {
```

Comment:
For clarity, it may be better to have these as static final constants at the top of the class ( RADIUS and MAX_RESULTS).